evalml/tests/automl_tests/test_automl.py

```diff
@@ -1383,7 +1383,7 @@ class Pipeline2(DummyPipeline):
     elif problem_type_value == ProblemTypes.TIME_SERIES_REGRESSION:
         automl = AutoMLSearch(X_train=X, y_train=y, problem_type=problem_type_value, max_iterations=3,
                               allowed_pipelines=[Pipeline1, Pipeline2], objective=objective,
-                              additional_objectives=[], problem_configuration={'gap': 0, 'max_delay': 0}, train_best_pipeline=False, n_jobs=1)
```

Comment:
Can we keep `train_best_pipeline=False`?

Reply:
+1